src/lib/core/add-alias.ts

```diff
@@ -14,7 +14,7 @@ import {extendObject} from '../utils/object-extend';
  * and suffix (if available).
  */
 export function mergeAlias(dest: MediaChange, source: BreakPoint | null): MediaChange {
-  return extendObject(dest, source ? {
+  return extendObject(dest || {}, source ? {
```

Comment:
Is this needed?

Reply:
perhaps not, but adds runtime safety without undue perf impact.